packages/vuelidate/src/core.js

```diff
@@ -670,11 +672,11 @@ export function setValidations ({
    * Executes the validators and returns the result.
    * @return {Promise<boolean>}
    */
-  function $validate () {
-    return new Promise(async (resolve) => {
-      if (!$dirty.value) $touch()
-      // await the watchers
-      await nextTick()
+  async function $validate () {
+    if (!$dirty.value) $touch()
+    // await the watchers
+    await nextTick()
+    return new Promise((resolve) => {
```

Comment:
Babel turns `async` functions into a hard-to-debug mess, try at https://babeljs.io/en/repl with env-preset's target set to `default`. I know that IE11 isn't longer a part of >1% but there are those who support it still and if that's the first entryy of async/await in the project, I would suggest restoring the old Promise wrapper you had there.

![image](https://user-images.githubusercontent.com/6743076/131412717-33f318b4-4fb0-44a8-a143-c429e65c7ba8.png)

Reply:
Sorry about that, but using async/await inside a promise callback was not ideal as well, throwing eslint errors.